content/en/docs/instrumentation/erlang/_index.md

```diff
@@ -18,7 +18,7 @@ The current status of the major functional components for OpenTelemetry Erlang/E
 
 | Traces  | Metrics | Logs    |
 | ------- | ------- | ------- |
-| Stable  | Alpha   | Not Yet Implemented |
+| [Stable][]  | [Stable][]   | Not Yet Implemented |
```

Comment:
To stay consistent with other language's pages please. For now, remove the link to the definition of stable. Let's have this in a separate issue, if we want to have this for *all* languages, cc: @cartermp